### PR TITLE
#172 Integrating the 0.0.7 release of the GeoConcerns Gem

### PR DESCRIPTION
**Pull request by jrgriffiniii**
First attempt at updating the Vagrant Box for the [0.0.7 release of the GeoConcerns Gem](https://rubygems.org/gems/geo_concerns/versions/0.0.7), as well as beginning to automate the deployment of the Rails App during the provisioning of the environment.

### CODE REVIEW

File: install_scripts/geo-concerns.sh [hunk 1]
```diff
@@ -2,20 +2,16 @@
 
 echo "Creating a stock Geo Concerns app"
 
-# create a curation-concerns app
+# Create a curation-concerns app
 rails new geo-concerns-demo --skip-spring
 cd geo-concerns-demo
 sudo gem install simple_mapnik
```
Comment (drh-stanford) [inline install_scripts/geo-concerns.sh]: this shouldn't be needed anymore, right?

File: install_scripts/ruby.sh [hunk 2]
```diff
@@ -6,11 +6,7 @@ echo "Setting up ruby environment"
 PACKAGES="imagemagick libreadline-dev libyaml-dev libsqlite3-dev libqtwebkit-dev nodejs zlib1g-dev libsqlite3-dev nodejs redis-server"
 sudo apt-get -y install $PACKAGES
```
Comment (drh-stanford) [inline install_scripts/ruby.sh]: some of these guys seem a little hidden in the "ruby" scriptie -- maybe put in geo_concerns?

File: install_scripts/geo-concerns.sh [hunk 3]
```diff
@@ -24,5 +20,5 @@ yes | cp -rf $SHARED_DIR/install_scripts/Rakefile .
 # Make solr core nonpersistent. Causes errors when solr-wrapper is restarted.
 sed -i 's/persist: true/persist: false/g' .solr_wrapper
 
-# setup fedora and solr
-rake demo:setup
+# Run the servers
+rake demo:servers
```
Comment (eliotjordan) [inline install_scripts/geo-concerns.sh]: Pretty sure we want to convert this back to `rake demo:setup`. The user needs to ssh into the box run a rake task to get everything going.

**Comment by eliotjordan** [comment]
Not able to get this running. I get this error in the fcrepo-webapp section. And then the servers shutdown.

```
[main] INFO org.eclipse.jetty.webapp.StandardDescriptorProcessor - NO JSP Support for /, did not find org.apache.jasper.servlet.JspServlet
```

**Comment by eliotjordan** [comment]
@jrgriffiniii This is good. I'm going to merge it.

**Pull request merged**